### ISSUE TITLE
Issue with modpack installation.

### ISSUE THREAD

**Issue by Alicazation** [cdbbnnyCode/modpack-installer#39]
After running `python install.py Better+MC+[FABRIC]+1.20.1+v19.zip` It failed with:

```
Traceback (most recent call last):
  File "install.py", line 33, in <module>
    from distutils import copy_tree
ImportError: cannot import name 'copy_tree'

```
I run ubuntu 16.04 and have python 3.6. I also have Oracle Java 17 console and shell.

**Comment by Alicazation** [comment]
Resolved: Updated to Python 3.8. I read install.py and in line 26/27 it says this
shutil.copytree doesn't accept dirs_exist_ok until 3.8 which is fairly modern (I think some LTS distros still use 3.6)
fall back to distutils copy_tree (the old solution) if needed

**Comment by cdbbnnyCode** [comment]
Oh... I did add some logic to use `distutils` on Python versions below 3.8, and it turns out I never tested it.

Thanks for the report!

**Comment by cdbbnnyCode** [comment]
Should be fixed in ac47ae2.
